downstream/modules/platform/proc-edge-manager-k8s-auth.adoc

```diff
@@ -0,0 +1,42 @@
+[id="edge-manager-k8s-auth"]
+
+= Authorizing with Kubernetes
+
+// Needs intro text
```

Comment:
"To authorize resources with Kubernetes while using the RHEM, you need to set variables in your Helm `values.yaml` file that fit your environment."
Or something like that, a bit more information about why these are needed would be good